js/UrbanAirship.js

```diff
@@ -494,6 +494,31 @@ class UrbanAirship {
     return UrbanAirshipModule.isQuietTimeEnabled();
   }
 
+  /**
+  * Sets Autobadging on iOS, either to true or false. Badging is not supported for Android.
+  * 
+  * @param {boolean} enabled Whether or not to enable Autobadging.
+  */
```

Comment:
Need an extra space at the beginning of the line.

Reply:
Okie doke! So add a space after the */ on 501?

Reply:
Before. The asterisks should all line up, like in the next method.

Reply:
Ah okay! I see it now! Thank you!